libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -590,5 +590,5 @@ class AC_AttitudeControl {
         float yaw_rate_cds;
         HeadingMode heading_mode;
     };
-    void input_thrust_vector_heading(const Vector3f& thrust_vector, HeadingCommand heading);
+    virtual void input_thrust_vector_heading(const Vector3f& thrust_vector, HeadingCommand heading);
```

Comment:
You should not need this one either, it just calls down to the other methods. `input_thrust_vector_rate_heading`, `input_thrust_vector_heading` which you have overridden.

Reply:
fixed.